Makefile

```diff
@@ -115,7 +115,10 @@ log:
 tail:
 	@echo "🥫 Reading logs (Apache2, Nginx) …"
 	tail -f logs/**/*
-
+  
+cover:
+	@echo "🥫 running …"
+	${DOCKER_COMPOSE_TEST} run --rm backend perl -MDevel::Cover tests/unit/*.t
```

Comment:
instead of a cover target, I would make using Cover the default (or maybe we could add the argument based on an env variable) ?

So I would add the ` HARNESS_PERL_SWITCHES="-MDevel::Cover"` to `unit_test` and `integration_test` targets (see https://github.com/openfoodfacts/openfoodfacts-server/pull/6926#discussion_r925607141)

Also, good to know, in the pull_request we could export the html report of cover (there is a notion of asset producted by a job). So it would be easy to get a branch report.

Reply:
> Also, good to know, in the pull_request we could export the html report of cover (there is a notion of asset producted by a job). So it would be easy to get a branch report.

It may be easier if we could export to codecov, but that's just my opinion. We could also export both of them!